lib/mirage.ml

```diff
@@ -2380,7 +2380,11 @@ let configure_opam t =
         | major::minor::_ ->
           let major = try int_of_string major with Failure _ -> 0 in
           let minor = try int_of_string minor with Failure _ -> 0 in
-          if (major, minor) >= (1, 2) then opam "install" ps else version_error ()
+          if (major, minor) >= (1, 2) then (
```

Comment:
Oh I'm sorry -- I thought the version check was elsewhere, but this also does that.  I just didnt want them to be duplicated.